src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -169,11 +179,49 @@ def generateBaseMesh(cls, region, options):
                         lNodeIds[n3][n2][n1] = nodeIdentifier
                         nodeIdentifier += 1
 
+            # Upper lobe nodes
+            for n3 in range(uelementsCount3 + 1):
+                reset = 0
+                uNodeIds.append([])
+                for n2 in range(uelementsCount2 + 1):
+                    uNodeIds[n3].append([])
+                    for n1 in range(uelementsCount1 + 1):
+                        uNodeIds[n3][n2].append(None)
+                        if (n1 == 0 or n1 == uelementsCount1) and (n2 == 0 or n2 == uelementsCount2):
```

Comment:
Prefer brackets so reader doesn't need to care about precedence rules:
``if ((n1 == 0) or (n1 == uelementsCount1)) and ((n2 == 0) or (n2 == uelementsCount2)):``
There are a few other places as well.